src/Common/utils.ts

```diff
@@ -96,10 +97,10 @@ const utils = {
   /**
    * Decode a `ClassicAddress` from a given x-address.
    *
-   * @see https://xrpaddress.info/
+   * @see {@link https://xrpaddress.info/|XRPL X-Address Format}.
    *
    * @param xAddress - The xAddress to decode.
-   * @returns A `ClassicAddress`
+   * @returns A `ClassicAddress`.
```

Comment:
We require full sentences (Start with a capital letter, end with a period, more or less).